app/ts/background/simulationModeHanders.ts

```diff
@@ -66,16 +66,17 @@ export async function sendTransaction(
 
 		const parentBlock = await block
 		if (parentBlock.baseFeePerGas === undefined) throw new Error(CANNOT_SIMULATE_OFF_LEGACY_BLOCK)
+		const inputAndDataMerged = 'data' in transactionDetails && transactionDetails.data !== undefined ? transactionDetails.data : 'input' in transactionDetails && transactionDetails.input !== undefined ? transactionDetails.input : new Uint8Array()
```

Comment:
Consider splitting this for readability.
```suggestion
		const inputAndDataMerged = 'data' in transactionDetails && transactionDetails.data !== undefined
			? transactionDetails.data
			: 'input' in transactionDetails && transactionDetails.input !== undefined
				? transactionDetails.input
				: new Uint8Array()
```